movies/src/resources/doc.json

```diff
@@ -0,0 +1,12 @@
+{
+  "@context": "/context.jsonld",
+  "@id": "/doc",
+  "@type": "hydra:ApiDocumentation",
```

Comment:
API documentation should provide entry point.

Reply:
ok